### PR TITLE
docs: updated the tour of heroes doc file

### PR DESCRIPTION
**Pull request by alkavats1**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: aio/content/tutorial/tour-of-heroes/toh-pt1.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ To automate that data flow, set up a two-way data binding between the `<input>`
 
 ### Two-way binding
 
-Refactor the details area in the `HeroesComponent` template so it looks like this:
+Refactor the details area in the `HeroesComponent` template, so it looks like this:
```
Comment (jessicajaniuk) [inline aio/content/tutorial/tour-of-heroes/toh-pt1.md:123]: Usually a comma would come before "so" when it's separating thoughts. However, in this case it's instructional. I don't think this comma is needed.

File: aio/content/tutorial/tour-of-heroes/toh-pt6.md [hunk 2]
```diff
@@ -167,7 +167,7 @@ The following `handleError()` method reports the error and then returns an innoc
 
 #### `handleError`
 
-The following `handleError()` can be shared by many `HeroService` methods so it's generalized to meet their different needs.
+The following `handleError()` can be shared by many `HeroService` methods, so it's generalized to meet their different needs.
```
Comment (jessicajaniuk) [inline aio/content/tutorial/tour-of-heroes/toh-pt6.md:170]: Same here. We're not separating thoughts. We're giving instructions. No need for a comma.

File: aio/content/tutorial/tour-of-heroes/toh-pt6.md [hunk 3]
```diff
@@ -385,7 +385,7 @@ The `$` is a convention that indicates `heroes$` is an `Observable`, not an arra
 <code-example header="src/app/hero-search/hero-search.component.html" path="toh-pt6/src/app/hero-search/hero-search.component.html" region="async"></code-example>
 
 Since `*ngFor` can't do anything with an `Observable`, use the pipe `|` character followed by `async`.
-This identifies Angular's `AsyncPipe` and subscribes to an `Observable` automatically so you won't have to do so in the component class.
+This identifies Angular's `AsyncPipe` and subscribes to an `Observable` automatically, so you won't have to do so in the component class.
```
Comment (jessicajaniuk) [inline aio/content/tutorial/tour-of-heroes/toh-pt6.md:388]: Same here. This is dependent clause and the comma is unnecessary.

**Review by jessicajaniuk — CHANGES_REQUESTED** [review]
@alkavats1 can you fix the commas here? For context: https://www.grammarly.com/blog/comma-before-so/

**Review by jessicajaniuk — APPROVED** [review]
@alkavats1 Thanks for the fixes! LGTM

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 2c5e5e49709c80c0380b3462aaabc75e81dd487c.

**Pull request closed without merging**